classes/Http/Controller/DashboardController.php

```diff
@@ -35,20 +35,29 @@ public function showSpeakerProfile()
     /**
      * Check to see if the CfP for this app is still open
      *
-     * @param integer $currentTime
+     * @param integer|\DateTimeInterface $currentTime
```

Comment:
It's a leftover from the current implementation. I wanted to keep that to not break stuff I didn't find. But you're right. I should remove that and actually break the code to find currently unchanged implementations.